scripts/traccc_throughput_mt_plotter.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python3
+#
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2023 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+#
+
+# Python import(s).
+import argparse
+import csv
+
+# ROOT import(s).
+import ROOT
+
+def runOnInputs(inputs, func):
+    '''Function running another function on every row in all input CSVs
+
+    Argument(s):
+       inputs  -- List of CSV file name, platform name pairs
+       func    -- The function-like-object to run on every CSV row
+    '''
+
+    # Loop over the files.
+    for input in inputs:
+        # Open this particular file.
+        with open(input[0], 'r') as csvFile:
+            csvReader = csv.DictReader(csvFile)
+            # Process each row of the file.
+            for row in csvReader:
+                # Execute the received function on this row.
+                func(row, input)
+                pass
+            pass
+        pass
```

Comment:
Why are all these pass statements here? :worried:

Reply:
Because most editors (at least Emacs and VSCode) handle these as "closing brackets". And know to indent the next line of the code "one less".

Reply:
Okay, sure, but let's not pollute the code with unnecessary nonsense to placate a few specific editors.